packages/init/legacy.js

```diff
@@ -0,0 +1,12 @@
+// legacy.js - call tolerant lockdown with default Agoric shims
+import { lockdown } from '@endo/lockdown';
+import './pre-remoting.js';
+
+// The loosest, non-stack-hiding SES environment.
```

Comment:
Not really loosest. I do not suggest making it looser; just altering the comment. Your point is that it is loose enough for your purposes. It's good you're not being looser than that.